canonical-websites/docs.snapcraft.io
# Upgrade to vanilla v1.8.0

- Use vanilla-framework `v1.8.0` instead of `v0.1.0`
- Squeeze content to `64.875rem`, like the current docs.snapcraft.io
- Copy some styling overrides and JavaScript for docs layouts from docs.vanillaframework.io (`_vf-overrides.scss`)
- Copy wholesale from snapcraft.io:
  - `_patterns_social-icons.scss`
  - `_snapcraft_footer.scss`
  - `_snapcraft_p-navigation.scss`
- Add https://highlightjs.org for syntax highlighting
- Add some custom styling:
  - For the navigation format as it comes from the forum (which uses `<h3>`s) 
  - For `blockquote`s, the spacing didn't look quite right
  - For `pre` and `code` so they play nicely with highlightjs
  - For the huge `.emoji` images that come from Discourse
- Rip out the old vf 0.1.0-based styles
- Write some JavaScript for adding `is-active` to the selected navigation item

Fixes #15
Fixes #5 

QA
--

`./run`, and check http://0.0.0.0:8029/ looks good and works well, even on small screens.
